src/components/players/playerDetails/mainStats.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import styles from './mainStats.module.scss';
+
+const MainStats = (props) => {
+    return (
+        <div className={styles.container}>
+            <div className={styles.title}>{props.title}</div>
+            <div className={styles.stat}>{props.stat}</div>
+        </div>
+    );
+}
```

Comment:
As you briefly mentioned before. It seems that the component for `MaintStats` and `AllStats` are exactly the same js. The only different is slight differences between the scss. 

Firstly, make one single component called `Stat` (notice the singular name)

Next, implement the css for this table:

<img width="444" alt="Screen Shot 2020-03-18 at 3 09 37 AM" src="https://user-images.githubusercontent.com/623755/76934674-f36bdd80-68c5-11ea-845c-fbc4afe42ce2.png">

Then in your `Stat` component, have a boolean prop called `isQuickStats`. If true, then adjust the css accordingly. I'll let you figure that one out. Let me know if you need clarification on this.